cinema/serializers.py

```diff
@@ -33,25 +33,45 @@ class Meta:
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
```

Comment:
As per task requirements:

> Image field should not be available on POST api/cinema/movies/.
> Image url should be shown on:
> Movie:  list and detail pages

With your implementation it is being shown on every endpoint. Make sure you show it only on List/Detail